data/human/weapons.txt

```diff
@@ -142,6 +164,27 @@ outfit "Modified Blaster Turret"
 		"hull damage" 8.8
 	description "This is a turreted version of the Modified Blaster, which is an Energy Blaster modified for greater power at the cost of higher energy consumption and heat. These turrets are popular with pirates and others who are trying to cram as much firepower into their ships as possible."
 
+outfit "Twin Modified Blaster"
+	category "Guns"
+	cost 84000
+	thumbnail "outfit/twin mod blaster"
+	"mass" 12
+	"outfit space" -12
+	"weapon capacity" -12
```

Comment:
```suggestion
	"mass" 15
	"outfit space" -15
	"weapon capacity" -15
```

This puts it at the same weight as a heavy laser, with 26 less total DPS, but with more range, and less overall drain, which feels like an appropriate tradeoff. Especially considering the effective DPS of the heavy laser is likely to be even higher since it takes only a single frame to hit and is more accurate overall.